src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -46,16 +69,42 @@ export default {
 		};
 	},
 	async mounted() {
-		this.allFacets = await fetchLoanFacets(this.apollo);
+		this.allFacets = await fetchLoanFacets(this.apollo, true);
 		await this.fetchFilterData(this.flssLoanSearch);
-	},
-	computed: {
-		totalCount() {
-			return 0;
-		},
+		const { loans, totalCount } = await runLoansQuery(
+			this.apollo,
+			{ ...this.flssLoanSearch, ...this.loanSearchState },
+			FLSS_ORIGIN_CATEGORY
+		);
+		this.loans = loans;
+		this.totalCount = totalCount;
 	},
 	methods: {
-		updateQuickFilters() {
+		async updateQuickFilters(filter) {
```

Comment:
this will be cleaned up in future tickets, i got ahead of myself and got a head start 😃